packages/admin/src/pages/super-admin-dashboard/user/[id]/block-user.page.tsx

```diff
@@ -0,0 +1,81 @@
+import { FlexibleQuestionPageLayout } from 'gap-web-ui';
+import { GetServerSidePropsContext } from 'next';
+import QuestionPageGetServerSideProps from '../../../../utils/QuestionPageGetServerSideProps';
```

Comment:
Should we import this from gap-web-ui? I can't work out why we have recreated `QuestionPageGetServerSideProps` within the admin app

Reply:
because it's essentially untestable otherwise - can't mock anything as the whole package is pre-built by `rollup` before we ever get to touch it from our tests